lib/fluent/engine.rb

```diff
@@ -90,12 +90,12 @@ def configure(conf)
         $log.info "gem '#{spec.name}' version '#{spec.version}'"
       end
 
+      @root_agent.configure(conf)
```

Comment:
We must call `configure` before configuration dump because
we need to link between element and configure_proxy.